src/main/java/com/checkmarx/sdk/ShardManager/ShardManagerHTTPInterceptor.java

```diff
@@ -1,161 +1,159 @@
-package com.checkmarx.sdk.ShardManager;

```

Comment:
what is the change in this file?

Reply:
organize imports - 3 unused imports were removed